pyproject.toml

```diff
@@ -33,6 +33,8 @@ pygments = "^2.15.1"
 google-api-python-client = "^2.90.0"
 chispa = "^0.9.2"
 grpcio = ">=1.48.1"
+plotly = "^5.15.0"
+pyspark = "^3.4.0"
```

Comment:
@vjr pyspark is listed as dev dependency below

Reply:
@gengliangwang i have updated the PR - how does it look now? i hope the poetry dependencies are correct for pyspark to be installed by users running pip install for the next release.

when do you expect to release next version of pyspark-ai to be available for "pip install" users?

thank you for your time!

Reply:
@gengliangwang pardon me going back and forth but i have restored the pyspark entry to the runtime deps section because "pip show pyspark-ai" is not showing pyspark in the "requires" list unless i do this...

Reply:
I would suggest adding plotly only in this PR.
pyspark is optional dependency since it is large and people who use this lib probably already installed it. https://github.com/databrickslabs/pyspark-ai/blob/master/pyproject.toml#L39

Reply:
Done!